src/logic/basic.lean

```diff
@@ -81,6 +81,12 @@ attribute [symm] ne.symm
 
 lemma ne_comm {α} {a b : α} : a ≠ b ↔ b ≠ a := ⟨ne.symm, ne.symm⟩
 
+lemma eq_iff_eq_cancel_left (a b c : α) (h : b = c) :
```

Comment:
```suggestion
lemma eq_iff_eq_cancel_left {a b c : α} (h : b = c) :
```
This is an iff lemma.Maybe it's better stated as such, and then it could be `@[simp]`. Not sure about this though.

Reply:
Done